tool-testsuite/test/org/antlr/v4/test/tool/TestPerformance.java

```diff
@@ -415,9 +415,9 @@ public void compileJdk() throws IOException, InterruptedException, ExecutionExce
 		assertTrue("The JDK_SOURCE_ROOT environment variable must be set for performance testing.", jdkSourceRoot != null && !jdkSourceRoot.isEmpty());
 
         compileJavaParser(USE_LR_GRAMMAR);
-		final String lexerName = "JavaLexer";
```

Comment:
:exclamation: Please revert the style changes here. They don't follow the code formatting options used by this project.